pkg/chartutil/capabilities.go

```diff
@@ -65,6 +76,18 @@ func (kv *KubeVersion) String() string { return kv.Version }
 // Deprecated: use KubeVersion.Version.
 func (kv *KubeVersion) GitVersion() string { return kv.Version }
 
+func ParseKubeVersion(version string) (*KubeVersion, error) {
+	sv, err := semver.NewVersion(version)
+	if err != nil {
+		return nil, err
+	}
+	return &KubeVersion{
+		Version: "v" + sv.String(),
```

Comment:
semver does not have a leading `v` but I'm just matching the default KubeVersion which does have the leading `v`.